src/component/UserAccountPages/ForgetPwdPage/ForgetPwdPage.module.scss

```diff
@@ -41,7 +46,23 @@
   color: #ff0000;
 }
 
-.signPage-smallButton1 {
+.buttons {
+  width: 335px;
+  margin: 24px 0px 0px 0;
+  padding: 15px 87px 15px 86px;
+  border: solid 1px #979797;
+  background-color: #23036a;
+  font-family: Roboto;
+  font-size: 24px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.38px;
+  color: #ffffff;
+}
+
+.smallButton1 {
```

Comment:
css naming:

I think button1 & button2 naming is a bit difficult to maintain in the future.